# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1513001314 18000
#      Mon Dec 11 09:08:34 2017 -0500
# Node ID c652f26370631ef26e8b6d76ac7d0d6a3dd9bf04
# Parent  41cf6ee90cb62d462e7318450a6188a3f8c76a5d
use Qt::NoItemFlags instead of 0 or nullptr

* variable-editor-model.cc (variable_editor_model::flags):
Return Qt::NoItemFlags instead of 0.

* workspace-model.cc (workspace_model::columnCount):
Initialize return value to Qt::NoItemFlags instead of nullptr.

diff --git a/libgui/src/variable-editor-model.cc b/libgui/src/variable-editor-model.cc
--- a/libgui/src/variable-editor-model.cc
+++ b/libgui/src/variable-editor-model.cc
@@ -422,17 +422,17 @@ variable_editor_model::flags (const QMod
         {
           if (editor_type (idx) != sub_string)
             return QAbstractTableModel::flags (idx);
         }
       return QAbstractTableModel::flags (idx) | Qt::ItemIsEditable;
       //return requires_sub_editor(idx) ?  QAbstractTableModel::flags (idx) : QAbstractTableModel::flags (idx) | Qt::ItemIsEditable;
     }
 
-  return 0;
+  return Qt::NoItemFlags;
 }
 
 // private slots
 
 void
 variable_editor_model::received_data (int r, int c,
                                       const QString& dat,
                                       const QString& class_info,
diff --git a/libgui/src/workspace-model.cc b/libgui/src/workspace-model.cc
--- a/libgui/src/workspace-model.cc
+++ b/libgui/src/workspace-model.cc
@@ -92,17 +92,17 @@ int
 workspace_model::columnCount (const QModelIndex&) const
 {
   return m_columnNames.size ();
 }
 
 Qt::ItemFlags
 workspace_model::flags (const QModelIndex& idx) const
 {
-  Qt::ItemFlags retval = nullptr;
+  Qt::ItemFlags retval = Qt::NoItemFlags;
 
   if (idx.isValid ())
     {
       retval |= Qt::ItemIsEnabled;
 
       if (m_top_level && idx.column () == 0)
         retval |= Qt::ItemIsSelectable;
     }
